features/base/styles/_.web.js

```diff
@@ -0,0 +1,4 @@
+export * from './web';
+
+require('style!./styles/FontAwesome.css');
```

Comment:
I'm not sure about putting 3rd party stuff inside source code. What if we have it as npm module (https://www.npmjs.com/package/font-awesome) instead?